crates/trie/common/Cargo.toml

```diff
@@ -18,7 +18,8 @@ alloy-rlp = { workspace = true, features = ["arrayvec"] }
 alloy-trie.workspace = true
 alloy-consensus.workspace = true
 alloy-genesis.workspace = true
-
+alloy-rpc-types-eth.workspace = true
+alloy-serde.workspace = true
```

Comment:
we only need those if the eip1186 feature is enabled, so those should be optional?